docs/proposals/cloudhub-enhancement.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Authorization Enhancement of CloudCore Websocket API
+authors:
+  - "spambot000"
+approvers: {}
+creation-date: 2024-04-06
+last-updated: 2024-04-06
+
+---
+# Authorization Enhancement for CloudCore Websocket API
+
+## Motivation
+CloudCore is the bridge between edge nodes and api server. However, it cannot restrict the access to cluster resources for a specific node. To address this issue, it is necessary to make an enhancement on CloudCore websocket API.
+
+#### Goal
+- Alpha
+
+Support node authorization mode. CloudCore can restrict an edge node from operating the resources owned by other edge nodes.
+- Beta
+
+Support RBAC authorization mode. KubeEdge users can utilize RBAC configurations to limit the access to their custom resources.
```

Comment:
It would be better to add the version in which feature would be implemented. For example, Alpha features will be implemented in version 1.17, and Beta features will be implemented in version 1.18.